apps/zk-explorer/components/Header.tsx

```diff
@@ -0,0 +1,62 @@
+'use client';
+
+import { Input, Dropdown, DropdownBasicButton, DropdownBody, MenuItem, Typography, Breadcrumbs, BreadcrumbsItem } from '@webb-tools/webb-ui-components';
+import { ThreeDotsVerticalIcon, Search, CheckboxBlankCircleLine } from '@webb-tools/icons';
+import { FC } from 'react';
+import { GitHubOAuthButton } from './GitHubOAuthButton';
+
+export const Header: FC<unknown> = () => {
+  const githubOAuthClientId = process.env.NEXT_PUBLIC_GITHUB_CLIENT_ID;
```

Comment:
in next.config.js, follow this [code](https://github.com/webb-tools/webb-dapp/blob/develop/apps/hubble-stats/next.config.js#L15) to make NEXT_PUBLIC_GITHUB_CLIENT_ID accessible to the client